avm-transpiler/src/transpile.rs

```diff
@@ -179,6 +179,17 @@ pub fn brillig_to_avm(
                     ],
                 });
             }
+            BrilligOpcode::Not { destination, source, bit_size } => {
```

Comment:
LGTM!

Reply:
Discussed on Slack: this will sadly break if working on U1, because we still don't have U1 (@dbanks12). One fix is to add a ToRadixLe with radix 1 after this for the U1 case (i.e., a truncating cast to U1). Alvaro might do this and we remove it later when we have U1.